tests/conftest.py

```diff
@@ -1347,8 +1348,13 @@ def finalizer():
             )
         if thread:
             thread.join()
+        log_path = ocsci_log_path()
         for worker in helpers.get_worker_nodes():
             if os.path.exists(f"/tmp/{worker}-top-output.txt"):
+                copyfile(
+                    f"/tmp/{worker}-top-output.txt",
```

Comment:
can you change '/tmp' reference to tempfile.mkdtemp(prefix='perf_')

Reply:
@vasukulkarni added code just to move the file from local /tmp to logs_dir. For /tmp/ reference change it required to be changed in lot of places, like fixture in conftest.py and 2 memory_leak related functions in helpers.

Anyway I have plans to fix issue https://github.com/red-hat-storage/ocs-ci/issues/811 during this fix will handle this change.